stdlib/string.gr

```diff
@@ -1604,18 +1489,11 @@ export let decodeRangeKeepBom = (bytes: Bytes, encoding: Encoding, start: Number
   decodeRangeHelp(bytes, encoding, false, start, size)
 }
 
-@disableGC
+@unsafe
 let rec decodeHelp = (bytes: Bytes, encoding: Encoding, skipBom: Bool) => {
   let bytesPtr = WasmI32.fromGrain(bytes)
   let bytesSize = WasmI32.load(bytesPtr, 4n)
-  Memory.incRef(WasmI32.fromGrain(decodeRangeHelp))
```

Comment:
I'm still sort of confused about how these incRefs can be dropped now. @ospencer @jozanza , maybe this will help me understand: am I correct in guessing that, with these changes, `--compilation-mode=runtime` still requires these sorts of manual incref/decref calls?

Reply:
Yup, that's correct. Happy to talk more about the implementation as well!